src/base/bittorrent/bencoderesumedatastorage.cpp

```diff
@@ -92,7 +92,7 @@ namespace
 BitTorrent::BencodeResumeDataStorage::BencodeResumeDataStorage(const Path &path, QObject *parent)
     : ResumeDataStorage {parent}
     , m_resumeDataPath {path}
-    , m_ioThread {new QThread {this}}
+    , m_ioThread {new QThread(this)}
```

Comment:
off topic
What is rule for deciding which form to use now? I can't see a pattern here.

Reply:
> What is rule for deciding which form to use now?

When we discussed this last time, we agreed that in the case of a constructor call, curly braces should be used only in extreme cases, namely, either when it is necessary to eliminate ambiguity, or when the list of parameters follows directly after the variable name:
```c++
Type name {param1, param2};
// but
Type *name = new Type(param1, param2);
```